lib/similarity_utils.py

```diff
@@ -0,0 +1,13 @@
+def pokrycie_przedzialow(przedzial1: list[int], przedzial2: list[int]) -> float:
+  ''' metoda przyjmuje jako argument dwie listy w postaci przedzialu liczbowego, tj. lista = [a, b], gdzie a <= b
+  metoda zwraca liczbę z przedziału [0, 1] '''
+  if len(przedzial1) != 2 or len(przedzial2) != 2:
+    return 0
+  elif przedzial1[1] <= przedzial1[0] or przedzial2[1] <= przedzial2[0]:
+    return 0
```

Comment:
To powinno się wywalić. Zastąpmy to asercjami. Podanie przedziału jako coś innego niż poprawny przedział nie powinno przejść tak samo jak jakiś poprawny przypadek

Reply:
Zamienione na asserty z odpowiednimi komentarzami